src/Icon/__snapshots__/Icon.spec.js.snap

```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Icon /> renders 1`] = `
+<DocumentFragment>
+  .c0 {
+  color: inherit;
+  font-size: inherit;
+}
+
+<i
+    class="mdi mdi-message re-icon c0"
+    name="message"
+  />
+</DocumentFragment>
+`;
+
+exports[`<Icon /> renders with passed class name 1`] = `
+<DocumentFragment>
+  .c0 {
+  color: inherit;
+  font-size: inherit;
+}
+
+<i
+    class="bagels bagels mdi mdi-message re-icon c0"
```

Comment:
I have no idea why `bagels` is being duplicated here. Logging out the output of `utils#getComponentClassName` shows the correct name but when the component is rendered, this duplicate class name is added 🤷‍♂️

Reply:
Custom class names are indeed duplicated when rendered:

<img width="425" alt="Screen Shot 2019-03-27 at 12 47 19 PM" src="https://user-images.githubusercontent.com/14184138/55107414-7d57e080-508e-11e9-856c-ec9316a0efdf.png">